src/ResourceManager/Search/Commands.Management.Search/help/Get-AzureRmSearchAdminKeyPair.md

```diff
@@ -0,0 +1,147 @@
+---
+external help file: Microsoft.Azure.Commands.Management.Search.dll-Help.xml
+Module Name: AzureRM.Search
+online version:
```

Comment:
Please add the online version.  Should be format: https://docs.microsoft.com/en-us/powershell/module/azurerm.search/<cmdletname>, in this case https://docs.microsoft.com/en-us/powershell/module/azurerm.search/get-azurermsearchadminkeypair

Apply everywhere.